src/broker/cas.c

```diff
@@ -1277,6 +1282,13 @@ cas_main (void)
 #if !defined(WINDOWS)
 		signal (SIGUSR1, query_cancel);
 #endif /* !WINDOWS */
+		cub_ddl_log_app_name (APP_NAME_CAS);
+		cub_ddl_log_br_name (shm_appl->broker_name);
+		cub_ddl_log_br_index (shm_as_index);
+		cub_ddl_log_db_name (db_name);
+		cub_ddl_log_user_name (db_user);
+		cub_ddl_log_ip (client_ip_str);
+		cub_ddl_log_pid (getpid ());
 		fn_ret = process_request (client_sock_fd, &net_buf, &req_info);
```

Comment:
Put an intentional blank line for readability.

```
cub_ddl_log_pid (getpid ());

fn_ret = process_request (client_sock_fd, &net_buf, &req_info);
```

Reply:
Thank you, i will change code as your recommend.